### DISCUSSION TITLE
Set of Pre-Value not recognized

### DISCUSSION THREAD

**Discussion by friedpa** [jomjol/AI-on-the-edge-device#1431] [General]
### The Problem

Nachdem aus irgendeinem Grund der Pre-Value außerhalb des Zeitfensters geraden ist, habe ich diesem auf den letzen RAW Wert gesetzt. Im Pre-Value tauchen wieder 6 Stellen anstatt der 4 Stellen. Unten links wird aber der RAW Wert noch immer mit dem vorherigen Pre-Value verglichen.

### Version

12.0.1

### Screenshots

![grafik](https://user-images.githubusercontent.com/42890695/199014675-ddebd354-55f7-4ce1-ad0a-0b930870312f.png)

**Comment by aberfisch** [comment]
> ### The Problem
> Nachdem aus irgendeinem Grund der Pre-Value außerhalb des Zeitfensters geraden ist, habe ich diesem auf den letzen RAW Wert gesetzt. Im Pre-Value tauchen wieder 6 Stellen anstatt der 4 Stellen. Unten links wird aber der RAW Wert noch immer mit dem vorherigen Pre-Value verglichen.
> 
> ### Version
> 12.0.1
> 
> ### Expected Behavior
> _No response_
> 
> ### Screenshots
> ![grafik](https://user-images.githubusercontent.com/42890695/199014675-ddebd354-55f7-4ce1-ad0a-0b930870312f.png)
> 
> ### Additional Context
> _No response_

+1 same here, I recognized it today and the shift was 1m3. Had the same behavior with the set-previous value and only after reboot the error was corrected.

**Comment by friedpa** [comment]
One more example. Just compare Previous Value in the Main and in the Error Window. Secondly where is the 351.8262"94" coming from? I only inserted 4 digits from the dot to the right...

![grafik](https://user-images.githubusercontent.com/42890695/200528148-3535a620-8c25-4122-8df3-6861705ee908.png)

**Comment by caco3** [comment]
@friedpa Nimmst Du bewusst ein Model, welches die Zwischenwerte nicht interpolieren kann (`N` in der Value)?

Ob 4 oder 6 Stellen ist nicht so relevant, vermutlich wird es in der Darstellung nicht konsistent gerunded.

> Unten links wird aber der RAW Wert noch immer mit dem vorherigen Pre-Value verglichen.

Das wird vermutlich erst nach der nächsten Runde korrekt sein!

> Secondly where is the 351.8262"94" coming from? I only inserted 4 digits from the dot to the right...

Ev. Interpolation
